tools/perf-automation/Azure.Sdk.Tools.PerfAutomation/Rust.cs

```diff
@@ -60,10 +65,55 @@ public UtilEventArgs(string methodName, string[] methodParams)
             IDictionary<string, string> packageVersions,
             bool debug)
         {
+
             // just make sure we have the target directory cleaned up of previous results in case of a test issue in a previous test / run
             _targetResultsDirectory = await CleanupAsync(project);
             Directory.CreateDirectory(_targetResultsDirectory);
-            return (String.Empty, String.Empty, String.Empty);
+            _debug = debug;
+
+            string flavor = debug ? "debug" : "release";
+            _testCommand = $"--{flavor} --package {primaryPackage} --test perf ";
+            var result = await Util.RunAsync(_cargoName, $"build {_testCommand} --message-format=json", WorkingDirectory, log: false);
+            // Look for errors in the build.
+            if (result.ExitCode != 0)
+            {
+                return (result.StandardOutput, result.StandardError, null);
+            }
+
+            Console.WriteLine("Parsing build output to find test executables...");
+            string[] json_elements = result.StandardOutput.Split(new[] { Environment.NewLine }, StringSplitOptions.RemoveEmptyEntries);
```

Comment:
I knew new lines are platform-dependent for text files, but - perhaps a silly question - is the same true of stdout? I guess I never really thought much of it. IMO, I usually just split on "\n" in the few times I'd had to process stdout as text and expect new lines.

Reply:
I'm not worried about linux, which uses "\n" anyway. My understanding is that `WriteFile` in text mode in Windows (and, therefore, anything that uses it) translates "\n" in output streams to "\r\n" anyway. Then there's Mac which - though I'm not actually sure if it's true today with OSX/macOS - uses "\r" only.